RetroBar/Controls/Clock.xaml.cs

```diff
@@ -27,7 +27,6 @@ public DateTime Now
         }
 
         private readonly DispatcherTimer clock = new DispatcherTimer(DispatcherPriority.Background);
-        private readonly DispatcherTimer singleClick = new DispatcherTimer(DispatcherPriority.Input);
```

Comment:
It was intentionally added in #118. Is the lag too much?

Reply:
Yeah, at least unbearable for me, the clock flyout in the stock taskbar opens immediately!

Reply:
OK, given that single-click is probably used much more often than double-click anyway, I am good with removing it.